### PR TITLE
Update base image from xenial-20170802 to xenial-20181005

### PR DESCRIPTION
**Pull request by yykamei**
Update base image of devon_rex_base.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial-20170802
+FROM ubuntu:xenial-20181005
```
Comment (akito19) [inline Dockerfile]: [Question] Is 18.04 LTS too early to adopt for Sider?
https://hub.docker.com/_/ubuntu/

Comment (yykamei) [reply Dockerfile]: Yes, I thought it's early. For example, there is a problem that 18.04 LTS does not provide `python-software-properties.` Perhaps other problem will occur.  

However, this is the chance to update images.

What do you think? Should I update to 18.04?

Comment (akito19) [reply Dockerfile]: I support your idea. I also believe that you don't have to update Ubuntu version if a required software can't be installed into 18.04 image.

Comment (yykamei) [reply Dockerfile]: :ok: we will update to next LTS within three years!

**Pull request merged**